task-generator/trusted-artifacts/ta.go

```diff
@@ -334,6 +335,16 @@ func perform(task *pipeline.Task, recipe *Recipe) error {
 				},
 			},
 			Args: args,
+			ComputeResources: core.ResourceRequirements{
+				Requests: core.ResourceList{
+					core.ResourceCPU:    resource.MustParse("1"),
+					core.ResourceMemory: resource.MustParse("3Gi"),
+				},
+				Limits: core.ResourceList{
+					core.ResourceCPU:    resource.MustParse("1"),
+					core.ResourceMemory: resource.MustParse("3Gi"),
+				},
+			},
```

Comment:
Ideally the values here would be part of the recipe, nothing that would block this change, this can be added if/when there is a need to modify this in the future

Reply:
I agree (as far as I understand the architecture here) but that would require bigger code change and I did not wanted to go too deep to it for now. But, let me know and I will happily work on it in different PR.